packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.Props.ts

```diff
@@ -30,6 +30,11 @@ export interface IProgressIndicatorProps {
   percentComplete?: number;
 
   /**
+   * Text alternative of progress status
```

Comment:
Maybe something like "step 1" "step 2" "step 3" ? If that's the case, you should be clear in the comment that that's the only scenario in which is should be used.